### PR TITLE
MEN-5966 - access to jwt from the UI

### PR DESCRIPTION
**Pull request by mzedel**
Screenshot 2022-10-14 at 16 34 57

### CODE REVIEW

**Comment by merlin-northern** [comment]
@michaelatmender I beg you :)

**Comment by michaelatmender** [comment]
@merlin-northern I've tried, but I can't get this PR running on my local setup. Issues with the styles for me

**Review by michaelatmender — CHANGES_REQUESTED** [review]
As suspected, it was something on my local setup that caused me issues! 

PR looks fine, only thing I would suggest is to add `word-break: break-all` styles to `.inventory-text.tenant-token-text`.

File: src/js/components/settings/organization/organization.js [hunk 1]
```diff
@@ -48,6 +48,25 @@ export const OrgHeader = () => {
   );
 };
 
+export const CopyTextToClipboard = ({ token }) => {
+  const [copied, setCopied] = useState(false);
+  const { classes } = useStyles();
+
+  const onCopied = () => {
+    setCopied(true);
+    setTimeout(() => setCopied(false), 5000);
```
Comment (merlin-northern) [inline src/js/components/settings/organization/organization.js:57]: nit pick:magic number, could we have a const for that?

**Review by merlin-northern — APPROVED** [review]
lgtm! tested.

File: src/js/components/settings/organization/organization.js [hunk 2]
```diff
@@ -48,6 +48,25 @@ export const OrgHeader = () => {
...
+  return (
+    <div>
+      <CopyToClipboard text={token} onCopy={onCopied}>
+        <Button startIcon={<CopyPasteIcon />}>Copy to clipboard</Button>
+      </CopyToClipboard>
+      <div className={classes.copyNotification}>{copied && <span className="green fadeIn">Copied to clipboard.</span>}</div>
+    </div>
+  );
+};
+
```
Comment (merlin-northern) [inline src/js/components/settings/organization/organization.js:69]: nice refactoring!

**Comment by merlin-northern** [comment]
> @merlin-northern I've tried, but I can't get this PR running on my local setup. Issues with the styles for me:
> 
> `Refused to apply style from 'https://localhost/ui/main.css?b7f7bb004a078c11' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`
> 
> So the UI is not being rendered

no worries mate, I gave it a spin, looks good.

File: src/js/components/settings/organization/organization.js (hunk 1 above)
Comment (mzedel) [reply src/js/components/settings/organization/organization.js]: https://github.com/mendersoftware/gui/pull/3211/commits/4f079cb7dc04d6b31fa8d0c8c3bac7ae0a2eca04 <- should address this for... several locations... 🙈

**Review by michaelatmender — APPROVED** [review]

**Pull request merged**